packages/main/src/themes/MultiComboBox.css

```diff
@@ -2,6 +2,10 @@
 @import "./InputIcon.css";
 @import "./InvisibleTextStyles.css";
 
+:host {
```

Comment:
Why are you setting additional CSS var if it could only be 1 or 0?

Is there a reason why you can't just write the following CSS: 
```
:host {
	min-width: calc(var(--_ui5_input_min_width) + (var(--_ui5-input-icons-count)*var(--_ui5_input_icon_width)) 
}

:host[tokenizer-available] {
	min-width: calc(var(--_ui5_input_min_width) + (var(--_ui5-input-icons-count)*var(--_ui5_input_icon_width)) + var(--_ui5_input_tokenizer_min_width)));
}

Instead of the current:
:host {
	min-width: calc(var(--_ui5_input_min_width) + (var(--_ui5-input-icons-count)*var(--_ui5_input_icon_width)) + (var(--_ui5_input_has_tokens)*var(--_ui5_input_tokenizer_min_width)));
}
```